vision/emergent_object/emergent_object_processing.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
```

Comment:
Was this file meant to be in this PR?

Reply:
Yeah... I made the same mistake because I forgot that branches aren't always created from develop.